components/info/AvatarCircle.tsx

```diff
@@ -36,7 +35,7 @@ const AvatarCircle = ({ variant = 'info' }: Props) => {
 
   return (
     <View
-      className={clsx('flex items-center justify-center rounded-full p-4', {
+      className={clsx('flex items-center justify-center self-center rounded-full p-4', {
```

Comment:
I removed it few commits ago, because centering should be handled by parent component in my opinion.